apps/champions/lib/champions/battle.ex

```diff
@@ -7,19 +7,22 @@ defmodule Champions.Battle do
 
   alias GameBackend.Campaigns
   alias GameBackend.Campaigns.CampaignProgress
+  alias GameBackend.Units
   alias GameBackend.Users
 
   @doc """
   Plays a level for a user, which means fighting its units with their selected ones.
   Returns `:win` or `:loss` accordingly, and updates the user's progress if they win..
   """
   def fight_level(user_id, level_id) do
-    with {:user, {:ok, user}} <- {:user, Users.get_user(user_id)},
+    with {:user, {:ok, _user}} <- {:user, Users.get_user(user_id)},
```

Comment:
What's the reason of this `get_user()` here? Just checking that it exists?

Reply:
yes!

Reply:
But we have a new User.exists? function now so i'm going to replace this with that one